exercises/concept/party-robot/.docs/introduction.md

```diff
@@ -1,5 +1,25 @@
 # Introduction
 
+Go programs are organized into packages. 
+A package is a collection of source files in the same directory that are compiled together. 
+Functions, types, variables, and constants defined in one source file are visible to all other source files within the same package. 
+Similar to program languages like Java that use access modifiers such as `public` and `private`, Go determines if an item is exported and unexported through how it is declared. 
+To allow an item to be called by code in other packages, the name must start with a capital letter.
```

Comment:
This (line 7+) isn't needed to solve the exercise, and so I don't think we want to include it in the exercise really.

The intro is great here, but the public/private part should, I think, only really be in `about.md`

Reply:
Sounds good - added two commits for moving the public/private info from `party-robot/introduction.md` to `concept/packages/about.md`